runtime/src/main/scala/com/ing/baker/runtime/actor/process_index/ProcessIndex.scala

```diff
@@ -296,7 +296,7 @@ class ProcessIndex(cleanupInterval: FiniteDuration = 1 minute,
         case Some(processMetadata) if processMetadata.isDeleted => sender() ! ProcessDeleted(processId)
         case Some(processMetadata) =>
           getCompiledRecipe(processMetadata.recipeId) match {
-            case Some(compiledRecipe) => sender() ! RecipeFound(compiledRecipe)
+            case Some(compiledRecipe) => sender() ! RecipeFound(compiledRecipe, 0L)
```

Comment:
Is this  a problem, always returning timestamp `0L`?

Reply:
Fixed